streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/config/AlertRestTemplateConfig.java

```diff
@@ -28,8 +28,10 @@ public class AlertRestTemplateConfig {
    *
    * @return RestTemplate
    */
-  @Bean(name = "alertRestTemplate")
+  @Bean(name = BEAN_NAME_ALERT_REST_TEMPLATE)
```

Comment:
The content in `Annotations` can be more intuitive and readable by maintaining direct string references without extracting constants

Reply:
I have recovered